### PR TITLE
Add overview dashboard to cert manager integration

### PR DESCRIPTION
**Pull request by arapulido**
### What does this PR do?

It adds an overview dashboard to the cert manager integration

### Review checklist

- [X] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [X] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by arapulido** [comment]
Some screenshots from staging:

image

image

image

image

File: cert_manager/README.md [hunk 1]
```diff
@@ -4,15 +4,26 @@
...
 ## Setup
 
-Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
 
 ### Installation
 
+#### With Agent versions >=7.26.0 or >=6.26.0
+
+The best way to use an integration from integrations-extra with the Docker Agent is to build the Agent with this integration installed. Use the following Dockerfile to build an updated version of the Agent that includes the `cert_manager` integration from integrations-extras.
```
Comment (ruthnaebeck) [inline cert_manager/README.md:17]:
```suggestion
To use an integration from `integrations-extra` with the Docker Agent, Datadog recommends building the Agent with the integration installed. Use the following Dockerfile to build an updated version of the Agent that includes the `cert_manager` integration from `integrations-extras`:
```

File: cert_manager/README.md [hunk 2]
```diff
@@ -4,15 +4,26 @@
...
+![Cert-Manager Overview Dashboard][2]
+
 ## Setup
 
-Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
 
 ### Installation
 
+#### With Agent versions >=7.26.0 or >=6.26.0
```
Comment (ruthnaebeck) [inline cert_manager/README.md:15]:
```suggestion
#### Agent versions >=7.26.0 or >=6.26.0
```

File: cert_manager/README.md [hunk 3]
```diff
@@ -4,15 +4,26 @@
...
+
+The best way to use an integration from integrations-extra with the Docker Agent is to build the Agent with this integration installed. Use the following Dockerfile to build an updated version of the Agent that includes the `cert_manager` integration from integrations-extras.
+
+FROM gcr.io/datadoghq/agent:latest
+RUN agent integration install -r -t datadog-cert_manager==<INTEGRATION_VERSION>
+
+#### With Agent versions <7.26.0 or <6.26.0
```
Comment (ruthnaebeck) [inline cert_manager/README.md:22]:
```suggestion
#### Agent versions <7.26.0 or <6.26.0
```

File: cert_manager/assets/dashboards/certmanager_overview.json [hunk 4]
```diff
@@ -0,0 +1 @@
...","line_type":"solid","line_width":"normal"},"display_type":"bars"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]}},{"id":5992721770987361,"layout":{"x":49,"y":25,"width":60,"height":47},"definition":{"title_size":"16","title_align":"left","type":"log_stream","indexes":[],"query":"service:cert-manager*","sort":{"column":"time","order":"desc"},"columns":["host","service"],"show_date_column":true,"show_message_column":true,"message_display":"expanded-md"}},{"id":917472946219287,"layout":{"x":0,"y":16,"width":47,"height":8},"definition":{"type":"note","content":"ACME Client","background_color":"orange","font_size":"18","text_align":"center","vertical_align":"center","show_tick":true,"tick_pos":"50%","tick_edge":"bottom","has_padding":true}},{"id":8624676510666531,"layout":{"x":49,"y":16,"width":60,"height":8},"definition":{"type":"note","content":"Cert Manager Logs","background_color":"orange","font_size":"18","text_align":"center","vertical_align":"center","show_tick":true,"tick_pos":"50%","tick_edge":"bottom","has_padding":true}}],"template_variables":[],"layout_type":"free","is_read_only":false,"notify_list":[],"id":"wzm-6tm-d6b"}
```
Comment (ruthnaebeck) [inline cert_manager/assets/dashboards/certmanager_overview.json:1]: The description needs to be filled out. See an [example here](https://github.com/DataDog/integrations-core/pull/6806/files#diff-e0e3367a0193faa1667f5ef4a0a446db98c2109f3029439ffe8dd43643b53340R18).

Also, having this all on 1 line makes it harder to review. The PR linked above has an example.

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

**Review by ruthnaebeck — APPROVED** [review]
👍  for docs

**Review by hithwen — APPROVED** [review]

**Pull request merged**